packages/server/src/items/purchasable.ts

```diff
@@ -88,22 +88,22 @@ export class Purchasable {
     }
   }
 
-  static createStandFromItem(createStandFrom: Item, creator: Mob): boolean {
-    itemGenerator.createItem({
-      type: 'potion-stand',
-      subtype: createStandFrom.subtype,
-      position: creator.position,
-      attributes: {
-        templateType: createStandFrom.type,
-        items: 1,
-        capacity: 20
-      }
-    });
-
-    createStandFrom.destroy();
-
-    return true;
-  }
+  // static createStandFromItem(createStandFrom: Item, creator: Mob): boolean {
+  //   itemGenerator.createItem({
+  //     type: 'potion-stand',
+  //     subtype: createStandFrom.subtype,
+  //     position: creator.position,
+  //     attributes: {
+  //       templateType: createStandFrom.type,
+  //       items: 1,
+  //       capacity: 20
+  //     }
+  //   });
+
+  //   createStandFrom.destroy();
+
+  //   return true;
+  // }
```

Comment:
Since we aren't using this code, lets just remove it and not comment it out. It adds to the noise of the codebase if we leave it.